proposals/separate-standalone-driver/README.md

```diff
@@ -0,0 +1,190 @@
+
+High overhead: Launching a Kubernetes pod merely to execute a few API calls introduces significant latency. Often, the pod scheduling and startup time outweighs the driver's actual processing time.
+
+Resource availability problems: There's no guarantee the Kubernetes cluster has sufficient resources to schedule the driver pod. If scheduling fails, the pipeline gets stuck. The UI currently doesn't show driver pod scheduling failures, which makes it hard to debug and understand what's going on.
+
+## Current state details
+
+Let's take a look at the copy of [hello_world.yaml](hello_world.yaml)  generated by the argo compiler tests.
+
+Execution Order:
+1. entrypoint
+   Type: DAG
+The root DAG represents the entire pipeline run.
+2. root-driver (template: system-dag-driver)
+   Type: Container
+   Purpose: Initializes the root DAG. Creates an MLMD execution for the DAG.
+Outputs:
+execution-id – ID of the DAG execution (created during root-driver execution)
+
+Tasks inside:
+- hello-world-driver (template: system-container-driver)
+  Purpose: Check for the existence of an execution in the cache, if no prepare the MLMD execution of the hello-world container task, and generate the appropriate pod-spec-patch.
+Outputs:
+pod-spec-patch – patch for the system-container-executor pod; inserts the correct image and command for the main container. 
+cached-decision – if true, the next step will be skipped
+- hello-world (template: system-container-executor)
+   Depends on: hello-world-driver.Succeeded
+   Purpose: Executes the hello-world component
+Inputs:
+pod-spec-patch — patch for the pod generated in the previous step
+cached-decision — used as a skip condition
+The system-container-executor template defines the main container that runs the user-defined code.
+
+Overview of the Argo workflow node structure for the container-driver
+```yaml
+  templates:
+    - name: system-container-driver
+      container:
+        args:
+        ...
+        command:
+        - driver
+      image: ghcr.io/kubeflow/kfp-driver
+      outputs:
+        parameters:
+          - name: pod-spec-patch
+            valueFrom:
+              path: /tmp/outputs/pod-spec-patch
+          - name: cached-decision
+            valueFrom:
+              path: /tmp/outputs/cached-decision
+```
+It creates a pod that launches the driver container using the kfp-driver image.
+
+## Alternative
+
+Instead of launching a new driver's pod using a container template, configure the system to send requests to an already running server.
+Something like this:
+```yaml
+  templates:
+  - name: system-container-driver
+    request:
+        args:
+        ...
+      outputs:
+        parameters:
+          - name: pod-spec-patch
+            jsonPath: $.pod_spec_patch
+          - name: cached-decision
+            jsonPath: $.cached_decision
+```
+### Requirements:
+- Execute a remote call with parameters
+- Read the response
+- Extract parameters from the response 
```

Comment:
What component handles the parameter extraction and podspecpatch creation and application? The AWF controller?